options.go

```diff
@@ -76,7 +76,8 @@ type RequestOptions struct {
 
 // StreamRequestOptions are stream request related options
 type StreamRequestOptions struct {
-	Interval timeMillisFlag `long:"stream-interval" description:"Interval between consecutive stream message sends, applicable separately to every stream request opened on a connection."`
+	Interval             timeMillisFlag `long:"stream-interval" description:"Interval between consecutive stream message sends, applicable separately to every stream request opened on a connection."`
+	DelayCloseSendStream timeMillisFlag `long:"delay-close-send-stream" description:"Delay the closure of send stream once all the request messages have been sent."`
```

Comment:
We should add a validation with this option that it is not used with the stdin option (they are not compatible right?)

Reply:
It's good to keep this option available for interactive mode as well. This enables users to pipe a file to the `stdin` where the file reads EOF in the end, stream closure due to stdin EOF can be delayed with this option.